src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -65,11 +65,11 @@ function getFacilityPageKey(state) {
 }
 
 async function vaFacilityNext(state, dispatch) {
-  let eligibility = getEligibilityStatus(state);
+  let eligibility = selectEligibility(state);
```

Comment:
Consolidated an simplified the selector, so we have to update that in a few places.